controllers/tasks.js

```diff
@@ -0,0 +1,98 @@
+const Task = require('../models/Task')
+
+const addTask = (req, res) => {
+    res.render('pages/addTask')
+}
+
+const createTask = async (req, res) => {
+    try {
+        if (req.body.complete) {
+            req.body.completed=true
+        }
+        console.log(req.body, "Inside the create func")
+        await Task.create(req.body)
+        req.session.pendingMessage = "The task was created."
+        res.redirect('/tasks')
+    } catch (err) {
+        if (err.name === 'ValidationError') {
+            res.locals.message = Object.values(err.errors)
+                .map((item) => item.message)
+                .join(', ')
+        } else {
+            res.locals.message = "Something went wrong."
+        }
+        res.render('pages/addTask')
+    }
+}
+
+const deleteTask = async (req, res) => {
+    try {
+        const task = await Task.findByIdAndDelete(req.params.id)
```

Comment:
No need to declare the `task` variable if it's never later used. This line can just look like this:

```javascript
await Task.findByIdAndDelete(req.params.id)
```

Reply:
got it. Thanks